utils/pofix.py

```diff
@@ -175,6 +175,79 @@
 ("most important things about being a leader is....", "most important things about being a leader is..."),
 ),
 
+"wesnoth-sof" : (
+# fixes added in 1.16.2+dev
+("Fire. The sceptre has a long, glorious,", "Fire. The Sceptre has a long, glorious,"),
+("by Haldric the Great. Nor will I tell the story of Garard I, ", "by Haldric the Great, nor will I tell the story of Garard I "),
+("challenge he set, or that of Konrad, King ", "challenge he set, nor that of Konrad, King "),
+("You must come into the caves, and go to the city ", "You must come into the caves and go to the city "),
+("I will need five thousand silver in advance, to ", "I will need five thousand silver in advance to "),
+("make sure you are going to get it done on time.", "make sure you get it done on time."),
```

Comment:
For this one, the context is clear that it will take most of the campaign to get it done (it's the SoF itself), so I don't think the exact tense used matters.